client/keys/add.go

```diff
@@ -117,56 +118,66 @@ func RunAddCmd(ctx client.Context, cmd *cobra.Command, args []string, inBuf *buf
 		return err
 	}
 
-	if dryRun, _ := cmd.Flags().GetBool(flags.FlagDryRun); !dryRun {
```

Comment:
Removing this here allows for the confirmation message to be displayed and `--multisig` to be run with `--dry-run`.